frontend/public/module/k8s/docker.js

```diff
@@ -37,8 +37,8 @@ export const getContainerState = function(containerStatus) {
   }
 
   const stateKey = keys[0];
-  state.label = stateKey;
   _.assign(state, containerStatus.state[stateKey]);
+  state.label = _.includes(stateKey, 'terminated') && state.exitCode ? `${stateKey} with exit code ${state.exitCode}` : stateKey;
```

Comment:
This won't print exit code 0. Is that intentional? Otherwise you want something like

```suggestion
  state.label = _.includes(stateKey, 'terminated') && _.isFinite(state.exitCode)
    ? `${stateKey} with exit code ${state.exitCode}`
    : stateKey;
```